### PR TITLE
Add ability to search users by labels

### PR DESCRIPTION
**Pull request by mkalenska**

### CODE REVIEW

**Comment by ec** [comment]
closes #799

**Review by ec — APPROVED** [review]

File: app/api/v2/admin/users.rb [hunk 1]
```diff
@@ -225,6 +225,24 @@ def search(field, value)
...
+              { code: 401, message: 'Invalid bearer token' }
+            ]
+            params do
+              requires :key,      type: String, desc: 'Label key'
+              requires :value,    type: String, desc: 'Label value'
+              optional :page,     type: Integer, default: 1,   integer_gt_zero: true, desc: 'Page number (defaults to 1).'
+              optional :limit,    type: Integer, default: 100, range: 1..1000, desc: 'Number of users per page (defaults to 100, maximum is 1000).'
+            end
+            get '/search' do
+              users = User.joins(:labels).where(labels: { scope: 'private', key: params[:key], value: params[:value] })
```
Comment (mod) [inline app/api/v2/admin/users.rb]: I think we need to remove private scope from the search, label can be public
I think we can improve this api

**Review by mod — CHANGES_REQUESTED** [review]

File: app/api/v2/admin/users.rb (hunk 1 above)
Comment (ec) [reply app/api/v2/admin/users.rb]: then, should we add a new optional parameter to specify?
Cause mixing private and public is not so good idea

Comment (mod) [reply app/api/v2/admin/users.rb]: ok, sounds good but maybe /search is not the right name or maybe we could do more search possibilities ?

**Review by mod — CHANGES_REQUESTED** [review]

**Pull request merged**